guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -14,6 +14,18 @@ endif::[]
 ifdef::katello[]
 This migration takes time, so if you depend on container content and need minimal upgrade downtime, use this procedure to migrate data.
 
+* Optional: If the upgrade succeeded, you can delete the `/var/lib/pgsql/data-old/` directory:
++
+[options="nowrap" subs="attributes"]
+----
+# rm /var/lib/pgsql/data-old/
+----
++
+{Project} creates this directory for backup purposes during the upgrade. 
+If the upgrade fails or {Project} loses PostgreSQL data shortly after the upgrade, you can use this directory to restore your external PostgreSQL database.
++
+Note, however, that after several database updates, the backup becomes obsolete, and cannot be used for disaster recovery anymore.
+
```

Comment:
@jherrman Have you noticed that only a preview for foreman-katello has been created by the bot? https://github.com/theforeman/foreman-documentation/pull/3769/#issuecomment-2773182599 That's because you added your lines within an `ifdef::katello[]` block. Try inserting them after line 13 instead and that will ensure that your lines appear in builds for all flavors.

Reply:
Yikes, you're right, I wasn't careful enough about the conditionals, apologies. Will fix.